resources/user-agents/browsers/ie-mobile/iemobile-8-0.json

```diff
@@ -32,279 +32,7 @@
       },
       "children": [
         {
-          "match": "Mozilla/4.0 (compatible; MSIE 6.0; Windows CE; IEMobile?#MAJORVER#.*) T-Mobile_LEO*",
```

Comment:
When adding tests, I noticed that all of these patterns in here were catching a lot of useragents and marking them as "IEMobile 8.0" when they were in fact "IEMobile 8.12", so I moved all of these patterns to the `iemobile-8-12.json` file and just left a fallback pattern here.  I didn't see much in the way of "IEMobile 8.0" in our logs or in other test suites.